src/ui/public/vis/update_status.js

```diff
@@ -61,17 +75,36 @@ const getUpdateStatus = ($scope) => {
     return false;
   }
 
-  const time = $scope.vis.params.timeRange ? $scope.vis.params.timeRange : $scope.vis.API.timeFilter.getBounds();
-  const width = $scope.vis.size ? $scope.vis.size[0] : 0;
-  const height = $scope.vis.size ? $scope.vis.size[1] : 0;
-  return {
-    aggs: hasChangedUsingGenericHashComparison('aggs', $scope.vis.aggs),
-    data: hasDataChanged($scope.visData),
-    params: hasChangedUsingGenericHashComparison('param', $scope.vis.params),
-    resize: hasSizeChanged(width, height),
-    time: hasChangedUsingGenericHashComparison('time', time),
-    uiState: hasChangedUsingGenericHashComparison('uiState', $scope.uiState)
-  };
-};
+  const status = {};
```

Comment:
I wouldn't introduce a breaking change here.

Reply:
i wouldn't call this a micro optimization .... 
looking at the performance profiles of visualize app we can see that updateStatus takes from 30 - 80% of our visualization rendering (depending on the size of data)

also we make it clear in our docs that we might break plugins with minor versions as we don't have any (stable) API yet